src/test/scala/uk/gov/nationalarchives/tdr/api/service/TransferAgreementServiceSpec.scala

```diff
@@ -131,19 +114,37 @@ class TransferAgreementServiceSpec extends AnyFlatSpec with MockitoSugar with Ma
     val consignmentId = UUID.randomUUID()
     val consignmentStatusId = UUID.fromString("d2f2c8d8-2e1d-4996-8ad2-b26ed547d1aa")
     val statusType = "TransferAgreement"
-    val statusValue = "Completed"
+    val statusValue = "InProgress"
     val createdTimestamp = Timestamp.from(now)
 
     val mockResponse = Future.successful(ConsignmentstatusRow(consignmentStatusId, consignmentId, statusType, statusValue, createdTimestamp))
     when(consignmentStatusRepositoryMock.addConsignmentStatus(any[ConsignmentstatusRow])).thenReturn(mockResponse)
 
     val service = new TransferAgreementService(consignmentMetadataRepositoryMock, consignmentStatusRepositoryMock, fixedUuidSource, fixedTimeSource)
-    val result: ConsignmentstatusRow = service.addTransferAgreementStatus(consignmentId, statusValue).futureValue
+    val result: ConsignmentstatusRow = service.addTransferAgreementStatus(consignmentId).futureValue
 
     result.consignmentstatusid shouldBe consignmentStatusId
     result.consignmentid shouldBe consignmentId
     result.statustype shouldBe statusType
     result.value shouldBe statusValue
     result.createddatetime shouldBe createdTimestamp
   }
+
+  "updateTransferAgreementStatus" should "add the correct Transfer Agreement Status" in {
```

Comment:
Not sure this test is actually checking that the correct status has been added. Maybe use `ArgumentCaptor`s to check that the correct arguments are being sent to the mocks. See for example https://github.com/nationalarchives/tdr-consignment-api/blob/master/src/test/scala/uk/gov/nationalarchives/tdr/api/service/AntivirusMetadataServiceSpec.scala#L41 for example of using captors.